game-core/src/main/java/games/strategy/triplea/settings/ProtectedStringClientSetting.java

```diff
@@ -0,0 +1,44 @@
+package games.strategy.triplea.settings;
+
+import java.util.Optional;
+
+import games.strategy.security.CredentialManager;
+import games.strategy.security.CredentialManagerException;
+
+class ProtectedStringClientSetting extends StringClientSetting {
```

Comment:
Note that by making this a `String` setting, we're making it easy for a developer to accidentally leak protected values throughout the system, where they may land someplace we didn't intend.  That's why the `CredentialManager` interface strongly discourages the use of the `protect(String)` and `unprotectToString()` methods.  Just something to think about. :smile: